### ISSUE TITLE
p-calendar with date view and month or year navigator active stops keyboard navigation to days

### ISSUE THREAD

**Issue by markermate** [primefaces/primeng#8789]
```
[x] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Current behavior**
After changing the month to any other month you are unable to tab to days and select a day.

**Expected behavior**
While using only keyboard navigation I should be able to select and change a day after I have change

**Minimal reproduction of the problem with instructions**
https://stackblitz.com/edit/github-b6f2a2

While using only keyboard navigation
- Change the month to any other month then the default
- Attempt to change the date

**What is the motivation / use case for changing the behavior?**
It should be the same as using the month view as well as the component should be able to be used via only keyboard navigation.

* **Angular version:** 9.0.2

* **PrimeNG version:** 9.0.6 

* **Browser:**  Chrome  81.0.4044.122 | Edge  44.18362.449.0

**Comment by yigitfindikli** [comment]
Unable to replicate with PrimeNG 11.0.0-rc.1, if the issue persists please create a new ticket with a test case reproducing the issue e.g. stackblitz or a github repo and it will be reviewed by our team once again.
